frontend/src/containers/PublicPage.js

```diff
@@ -14,10 +13,7 @@ export class PublicPage extends Component {
     if (this.props.isUserProfile) {
       const profile = this.props.group;
       profile.canEditUser = canEditUser(this.props.session, profile)
-
       return <ProfilePage profile={ profile } />
-    } else if (this.props.showOldCollectivePage) {
-      return <PublicGroup />
     } else if (this.props.slug ) {
       return <Collective />
     } else {
```

Comment:
Is this 'else' needed anymore? would be good to remove that and make <collective> the default component.